app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-3.md

```diff
@@ -0,0 +1,161 @@
+    - text: Beta location analysis - phase 2
+      href: /manage-school-placements/beta-location-analysis-phase-2/
+    - text: Beta location analysis - phase 4
+      href: /manage-school-placements/beta-location-analysis-phase-4/
+---
+
+We investigated the value of adding an additional region to our sample to improve the overall representativeness.
+
+## Expanding the sample area
+
+We considered 2 approaches for expanding the sample area:
+
+1. Expand into an adjacent local area
+2. Expand into a separate rural region
+
+### Expanding into an adjacent local area
+
+The advantage of expanding into an adjacent local area is that it streamlines expansion by utilising onboarded provider-school networks.
+
+The drawback of expanding into an adjacent local area is that the surrounding areas are heavily urban, so there is limited improvement in rural-urban representativeness.
+
+### Expanding into a separate rural region
+
+Expanding into a separate rural region has the advantage of potentially diversifying and improving rural-urban classification representativeness.
+
+The drawback of expanding into a separate rural region is that it increases the resources required to onboard a new area.
+
+## Focus on expanding into an adjacent local area
+
+We decided to focus on expanding to an adjacent local area.
+
+Engaging with local ecosystems is critical for a successful beta. By engaging with an adjacent local area, we can expand within the same networks already using the service.
+
+Our strategy will target the networks of current provider partners (providers located in our beta sample region). This will help us target our extension into areas within existing service user networks and optimise our chances of success by building on proven relationships.
+
+If evidence suggests rural schools will interact differently with our service, future expansion into rural areas will be explored.
+
+## Birmingham-based providers expanded network
+
+In general, providers in Birmingham operate across a wide area. This will be partially influenced by how we have classified the location of SCITTs, as some regional providers have the most placement school partners in Birmingham.
+
+Upon investigating the potential of these areas to improve Birmingham's representativeness, we chose Sandwell as the most appropriate region. Staffordshire placement schools had similar characteristics to Birmingham.
+
+![Heatmap showing the number of placement schools in each local authority that hosted trainees from a provider located in Birmingham. The lighter the colour, the greater the number of placement schools.](birmingham-based-providers-expanded-network.png "Heatmap showing the number of placement schools in each local authority that hosted trainees from a provider located in Birmingham. The lighter the colour, the greater the number of placement schools.")
+
+| Local authority | Number of providers in the area | Number of placements schools in the area |
```

Comment:
I think 'Number of providers in the area' could confuse readers, could we use something like 'Number of Birmingham based providers operating in the area'

Reply:
We decided not to change the heading as it relates to the section.